02-electric/electric-section-four/electric-section-four.css

```diff
@@ -1,32 +1,347 @@
+.electric-section-four-part-two-subtitle,
+.electric-section-four-part-two-paragraph-container,
+.electric-section-four-part-two-paragraph,
+.electric-section-four-part-three-container,
+.electric-section-four-part-three-image-container,
+.electric-section-four-part-three-copy-container,
+.electric-section-four-part-three-subtitle,
+.electric-section-four-part-three-paragraph-container,
+.electric-section-four-part-three-paragraph{
+    border:var(--temporary-border)
+}
+
+.electric-section-four-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
+  
+}
+
+@media (min-width:500px) {
+  .electric-section-four-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
+.electric-section-four-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+  "electric-section-four-title"
+  "electric-section-four-part-one-container"
+  "electric-section-four-part-two-container"
+  "electric-section-four-part-three-container";
+  justify-content: center;
+  row-gap: var(--subsection-container-row-gap-mobile);
+  column-gap:none;
+  padding: var(--subcontainer-padding-mobile);
+  border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-four-subcontainer{
+    grid-template-columns: repeat(3,1fr);
+    grid-template-rows: auto 40px auto auto auto auto;
+    grid-template-areas:
+    "electric-section-four-title electric-section-four-title electric-section-four-title"
+    ". electric-section-four-part-two-container ."
+    "electric-section-four-part-one-container electric-section-four-part-two-container electric-section-four-part-three-container"
+    "electric-section-four-part-one-container electric-section-four-part-two-container electric-section-four-part-three-container"
+    "electric-section-four-part-one-container electric-section-four-part-two-container electric-section-four-part-three-container"
+    "electric-section-four-part-one-container . electric-section-four-part-three-container";
+    padding: var(--subcontainer-padding-desktop);
+    row-gap: var(--subsection-container-row-gap-desktop);
+    column-gap:var(--subsection-container-column-gap-desktop) ;
+  }
+}
+
+.electric-section-four-title {
+  grid-area:electric-section-four-title;
+  text-align: center;
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
 }
 
-.electric-section-four-container > *{
-    border: solid  1px black;
+@media (min-width: 500px) {
+  .electric-section-four-title {
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
 }
 
-.electric-section-four-title{
-    grid-column: 1/2;
-    text-align: center;
+/*****PART one******/
+
+.electric-section-four-part-one-container{
+  grid-area:electric-section-four-part-one-container;
+  display: grid;
+  grid-template-columns: 1fr;
```

Comment:
align-content: start;


add this to part-two container and part-three container as well

Reply:
done